ground/src/main/res/layout/capture_location_task_frag.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  ~ Copyright 2023 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <data>
+    <variable
+      name="viewModel"
+      type="com.google.android.ground.ui.datacollection.tasks.location.CaptureLocationTaskViewModel" />
+  </data>
+
+  <TextView
+    android:id="@+id/gps_location"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="10dp"
+    android:text="@{viewModel.locationUpdates}"
+    android:textAppearance="@style/TextAppearance.AppCompat.Large" />
```

Comment:
I think we should now be using the Material styles rather than AppCompat directly. You can see the available themes and styles here https://m2.material.io/develop/android/theming/typography

Reply:
Done